src/core/core.scale.js

```diff
@@ -100,6 +103,22 @@ function computeTextSize(context, tick, font) {
 }
 
 module.exports = Element.extend({
+	/**
+	 * @private
+	 */
+	_configure: function() {
+		var me = this;
+		var opts = me.options;
+		var ttOpts = opts.tooltips || {};
+		var tickOpts = opts.ticks;
+
+		me._config = {
```

Comment:
As we add more options to this `_config` we might end up with quite a lot here and possibly have clashing names. It might be more organized to instead do something like:

```
me._config = {
  tooltips: {
    formatter: resolve([ttOpts.formatter, globalDefaults.tooltips.formatter, opts.formatter, globalDefaults.formatter]),
    callback: resolve([tickOpts.callback, tickOpts.userCallback]),
    formatter: resolve([tickOpts.formatter, opts.formatter, globalDefaults.formatter])
  }
}
```

Reply:
Its strictly private, so changing it later will not be a issue.